app/build.gradle

```diff
@@ -172,6 +172,53 @@ dependencies {
     implementation Deps.androidx_constraintlayout
     implementation Deps.androidx_preference
     implementation Deps.google_material
+
+
+    implementation 'com.squareup.okhttp3:okhttp:3.11.0'
+    implementation 'com.squareup.picasso:picasso:2.71828', {
```

Comment:
These don't seem to be needed right now? I think we should only add them once we do. They should also be `[android]TestImplementation`.